api_get.py

```diff
@@ -0,0 +1,66 @@
+from sanic import Sanic
+from sanic import response
+from serializers import *
+from functions import *
+from sanic.response import text
+import requests
+
+app = Sanic('some_name')
+
+
+# GET
+
+@app.route('/')
```

Comment:
We won't use views. Use class based view instead

Reply:
changed.